grunt/config/replace.js

```diff
@@ -37,6 +37,18 @@ module.exports = function (grunt, options) {
         if (configJson._xapi.hasOwnProperty('_activityID') && configJson._xapi._activityID == '') {
           grunt.log.writeln('WARNING: xAPI activityID has not been set');
         }
+
+        // cmi5 is a profile of the xAPI specification and some systems, e.g. SCORM Cloud
+        // do not work well with both manifest types, so remove the one which is not being used.
+        if (configJson._xapi.hasOwnProperty('_specification')) {
```

Comment:
try this on for size:
https://github.com/adaptlearning/adapt_framework/pull/1742